Tests/JWTKitTests/Utils/Wycheproof.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import XCTest
+
+func wycheproof(fileName: String, testFunction: (TestGroup) throws -> Void) throws {
+    let testsDirectory: String = URL(fileURLWithPath: "\(#file)").pathComponents.dropLast(2).joined(separator: "/")
```

Comment:
```suggestion
    let testsDirectory: String = URL(fileURLWithPath: "\(#filePath)").pathComponents.dropLast(2).joined(separator: "/")
```
Needed for compatibility with `.enableUpcomingFeature("ConciseMagicFile")`